fy22_cecil_b_moore/join_longform_to_user.sql

```diff
@@ -0,0 +1,5 @@
+create table longform_online as (
+select users.ip_address, users.responded_to_survey_question, users.q1, users.q2, users.q3, users.q4, users.q5_gender, users.q6, users.q7, longform."usage", longform.frequency, longform."mode", longform.mode_issues, longform.condition_1, longform.condition_2, longform.condition_3, longform.condition_4, longform.condition_5, longform.condition_6, longform.condition_7, longform.condition_8, longform.condition_9, longform.ideas, longform.priorities, longform.created_on 
```

Comment:
Use indentation / whitespace to tab-in any inner queries (instead of having everything fully untabbed to the left like this)